eden/scm/edenscm/ext/github/pull_request_body.py

```diff
@@ -48,19 +50,28 @@ def create_pull_request_title_and_body(
     owner, name = repository.get_upstream_owner_and_name()
     pr = pr_numbers_and_num_commits[pr_numbers_index][0]
     reviewstack_url = f"https://reviewstack.dev/{owner}/{name}/pull/{pr}"
+    if not template:
+        body = string.Template(f'''
+$commit_msg
+$horizontal_rule
+Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack]({reviewstack_url}). 
+$bulleted_list
+''')
+    else:
+        body = string.Template(template)
```

Comment:
This "works" as in doesn't crash at runtime but none of the template variables are injected properly yet. Any ideas what I am doing wrong?